cordapp-example/workflows-kotlin/src/main/kotlin/com/example/flow/PaybackFlow.kt

```diff
@@ -0,0 +1,128 @@
+package com.example.flow
+
+import co.paralleluniverse.fibers.Suspendable
+import com.example.contract.IOUContract
+import com.example.flow.PaybackFlow.Acceptor
+import com.example.flow.PaybackFlow.Initiator
+import com.example.state.IOUState
+import net.corda.core.contracts.*
+import net.corda.core.crypto.SecureHash
+import net.corda.core.flows.*
+import net.corda.core.transactions.SignedTransaction
+import net.corda.core.transactions.TransactionBuilder
+import net.corda.core.utilities.ProgressTracker
+import net.corda.core.utilities.ProgressTracker.Step
+
+
+/**
+ * This flow allows two parties (the [Initiator] and the [Acceptor]) to come to an agreement about the IOU encapsulated
+ * within an [IOUState].
+ *
+ * In our simple example, the [Acceptor] always accepts a valid IOU.
+ *
+ * These flows have deliberately been implemented by using only the call() method for ease of understanding. In
+ * practice we would recommend splitting up the various stages of the flow into sub-routines.
+ *
+ * All methods called within the [FlowLogic] sub-class need to be annotated with the @Suspendable annotation.
+ */
+object PaybackFlow {
+    @InitiatingFlow
+    @StartableByRPC
+    class Initiator(val loanTxId: String) : FlowLogic<SignedTransaction>() {
+        companion object {
+            object GENERATING_TRANSACTION : Step("Generating transaction based on IOU state.")
+            object VERIFYING_TRANSACTION : Step("Verifying contract constraints.")
+            object SIGNING_TRANSACTION : Step("Signing transaction with our private key.")
+
+            object FINALISING_TRANSACTION : Step("Obtaining notary signature and recording transaction.") {
+                override fun childProgressTracker() = FinalityFlow.tracker()
+            }
+
+            fun tracker() = ProgressTracker(
+                    GENERATING_TRANSACTION,
+                    VERIFYING_TRANSACTION,
+                    SIGNING_TRANSACTION,
+                    FINALISING_TRANSACTION
+            )
+        }
+
+        override val progressTracker = tracker()
+
+        /**
+         * The flow logic is encapsulated within the call() method.
+         */
+        @Suspendable
+        override fun call(): SignedTransaction {
+            // Obtain a reference to the notary we want to use.
+            val notary = serviceHub.networkMapCache.notaryIdentities[0]
+
+            // Stage 1.
+            progressTracker.currentStep = GENERATING_TRANSACTION
+            // Generate an unsigned transaction.
```

Comment:
Since this flow is called `Payback`, I assume that it's initiated by the `borrower`, below when you define the command you are saying that the required signature is `ourIdentity` meaning it's the `borrower`, but based on your contract; the required signature is that of the `lender`, so you should correct the below to become:
```kotlin
val txCommand = Command(IOUContract.Commands.Destroy(), lender.owningKey)
```